### PR TITLE
Add Sergei Pond's GB taxonomy tools.

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

File: packages/package_gb_taxonomy_8d245994d7/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="gb_taxonomy" version="8d245994d7">
+        <install version="1.0">
+            <actions>
+                <action type="shell_command">git clone https://github.com/spond/gb_taxonomy_tools.git gb_taxonomy</action>
+                <action type="shell_command">git reset --hard 8d245994d77546daac1f595b87a110a620426b4c</action>
+                <action type="shell_command">cmake -D INSTALL_PREFIX:PATH=$INSTALL_DIR .</action>
```
Comment (erasche) [inline packages/package_gb_taxonomy_8d245994d7/tool_dependencies.xml]: I seem to remember an IUC cmake dependency making it into the TS. Would you mind depending on that? https://github.com/galaxyproject/tools-iuc/pull/210

File: packages/package_gb_taxonomy_8d245994d7/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+categories:
+- Tool Dependency Packages
+description: Compiles and installs Sergei Pond's GenBank taxonomy tools.
+long_description: |
+  These are four simple utilities which perform certain manipulations and
+  visualization tasks on GenBank taxonomic information.
+name: package_gb_taxonomy_8d245994d7
+owner: iuc
```
Comment (erasche) [inline packages/package_gb_taxonomy_8d245994d7/.shed.yml]: `owner: iuc` but in tools-devteam?

**Comment by erasche** [comment]
:+1:

**Comment by jmchilton** [comment]
I know I have been very sporadic in mentioning this - but I feel like we should get serious about our own CONTRIBUTING policy https://github.com/galaxyproject/tools-devteam/blob/master/CONTRIBUTING.md. No new tools here - they should all go the the IUC.

I say this is the last one or we update the CONTRIBUTING file with a logical explanation for why Penn State pushes things here and the rest of the IUC pushes to tools-iuc.

**Comment by bgruening** [comment]
I can second this, I always wondering why we include new tools here.

**Comment by martenson** [comment]
Let's enforce it then.

**Comment by davebx** [comment]
Motion carries.

**Comment by davebx** [comment]
@bgruening Does github magic download a tarball of a specific changeset?

**Comment by bgruening** [comment]
Yes :) it's magic. E.g.: `curl -L https://github.com/user-or-org/repo/archive/sha1-or-ref.tar.gz | tar xz`

**Comment by bgruening** [comment]
@davebx I buy you a beer if you get this `GB_TAXONOMY_ROOT_PATH` automatically set during installation, so we do not net to specify it ;)

**Comment by davebx** [comment]
@bgruening Cool. Github magic it is then, give me a moment to figure it out. I'll get back to you on the taxonomy root path.

**Comment by bgruening** [comment]
it's always `'%s_ROOT_PATHs' % package_name.upper()`

**Comment by davebx** [comment]
... oh, that's what you meant. Ok, that's easy.

**Comment by davebx** [comment]
@bgruening https://github.com/galaxyproject/galaxy/pull/564

**Comment by davebx** [comment]
@bgruening And github magic works, https://github.com/galaxyproject/tools-devteam/pull/207

**Pull request merged**